screenpipe-audio/src/stt.rs

```diff
@@ -314,94 +302,130 @@ pub async fn stt(
 
     let audio_data = normalize_v2(&audio_data);
 
-    let frame_size = 1600; // 100ms frame size for 16kHz audio
-    let mut speech_frames = Vec::new();
-    let mut total_frames = 0;
-    let mut speech_frame_count = 0;
+    let project_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+
+    let embedding_model_path = project_dir
+        .join("models")
+        .join("pyannote")
+        .join("wespeaker_en_voxceleb_CAM++.onnx");
+
+    let segmentation_model_path = project_dir
+        .join("models")
+        .join("pyannote")
+        .join("segmentation-3.0.onnx");
+
+    let embedding_extractor = EmbeddingExtractor::new(
```

Comment:
any downside to load / unload the model at every chunk?

Reply:
This has been resolved